build.gradle

```diff
@@ -1,12 +1,12 @@
 buildscript {
-  ext.kotlin_version = '1.2.21'
+  ext.kotlin_version = '1.2.30'
```

Comment:
I'd just make the Kotlin update a separate PR. And it changes so frequently that maybe it's just worth it to wait until I get the updated Bookmarks feature in (since the version will no doubt change again in a week or two).

Reply:
Studio was complaining about it, but it was a warning, not an error.  It will still build with the old version, so I can revert.